### PR TITLE
New port: LOVE 3

### PR DESCRIPTION
**Pull request by JanTrueno**
# New Port for LOVE 3

## Game Information
- **Title**: LOVE 3
- **URL**: https://retroware.itch.io/love-3

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320 (no tester found)
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/love3/love3/gameinfo.xml
Comment (kloptops) [inline ports/love3/love3/gameinfo.xml:1]: There's two of these files?

Remove `ports/love3/love3/gameinfo.xml` :)

**Review by kloptops — CHANGES_REQUESTED** [review]
Delete the second gameinfo.xml, otherwise it looks good. 👍

File: ports/love3/love3/gameinfo.xml
Comment (JanTrueno) [reply ports/love3/love3/gameinfo.xml:1]: That one was still from Fred Wood, my bad :)

**Review by kloptops — DISMISSED** [review]
👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**